cpp/oneapi/dal/graph/detail/undirected_adjacency_vector_graph_impl.hpp

```diff
@@ -227,14 +227,26 @@ constexpr auto get_topology_vertex_degree(const topology<IndexType>& _topology,
     return _topology._degrees[vertex];
 }
 
+template <typename IndexType>
```

Comment:
here we use 'IndexType' while earlier I saw Index - align

Reply:
To align later after coding extension will be designed finally.